### PR TITLE
RDS Aurora MySQL guide

### PR DESCRIPTION
**Pull request by ewgenius**
- **rename rds-postgres quickstart**
- **Add RDS Aurora MySQL quickstart**

### CODE REVIEW

File: rds-aurora-mysql/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### Follow these steps to get started with AWS RDS Aurora (MySQL Compatible) as a Data Backend.
```
Comment (phillipleblanc) [inline rds-aurora-mysql/README.md]:
```suggestion
### Follow these steps to get started with federated SQL query against AWS RDS Aurora (MySQL Compatible).
```

Data Backend is our old name for a Data Accelerator, which is not what this is.

File: rds-postgresql/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-### Follow these steps to get started with AWS RDS as a Data Connector.
+### Follow these steps to get started with AWS RDS for PostgreSQL as a Data Backend.
```
Comment (phillipleblanc) [inline rds-postgresql/README.md]:
```suggestion
### Follow these steps to get started with federated SQL query against AWS RDS for PostgreSQL.
```

**Review by gloomweaver — APPROVED** [review]

File: rds-aurora-mysql/spicepod.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+version: v1beta1
+kind: Spicepod
+name: supabase_quickstart
```
Comment (lukekim) [inline rds-aurora-mysql/spicepod.yaml]: Update name

File: rds-aurora-mysql/spicepod.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+datasets:
+  - from: mysql:[remote_table_path]
+    name: [local_table_name]
+    params:
+      mysql_host: [db-host]
+      mysql_port: "[db-port]"
+      mysql_db: [db-name]
+      mysql_user: [username]
+      mysql_pass_key: [password]
+      mysql_sslmode: preferred
```
Comment (lukekim) [inline rds-aurora-mysql/spicepod.yaml]: Add comment as to the default. If this is the default, remove.

Comment (ewgenius) [reply rds-aurora-mysql/spicepod.yaml]: added

File: rds-aurora-mysql/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+### Follow these steps to get started with federated SQL query against AWS RDS Aurora (MySQL Compatible).
```
Comment (lukekim) [inline rds-aurora-mysql/README.md]: First line should be title. E.g. `# <title>`. Follow these steps should probably just be paragraph.

File: rds-postgresql/README.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
-### Follow these steps to get started with AWS RDS as a Data Connector.
```
Comment (lukekim) [inline rds-postgresql/README.md:1]: Same as other comment. First line should be `# `

File: rds-aurora-mysql/README.md (hunk 5 above)
Comment (ewgenius) [reply rds-aurora-mysql/README.md]: updated

File: README.md [hunk 7]
```diff
@@ -15,7 +15,8 @@ Designed to take 1-5 mins each to get started quickly!
 - [S3 Data Connector](./s3/README.md)
 - [Databricks Delta Lake Data Connector](./databricks/README.md)
 - [Supabase Data Connector](./supabase/README.md)
-- [AWS RDS Data Connector](./rds/README.md)
+- [AWS RDS PostgreSQL](./rds-postgresql/README.md)
```
Comment (lukekim) [inline README.md]: Suffix: Data Connector

File: README.md [hunk 8]
```diff
@@ -15,7 +15,8 @@ Designed to take 1-5 mins each to get started quickly!
...
+- [AWS RDS Aurora (MySQL Compatible)](./rds-aurora-mysql/README.md)
```
Comment (lukekim) [inline README.md]: Suffix: Data Connector

File: rds-aurora-mysql/README.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
+# AWS RDS Aurora (MySQL Compatible)
```
Comment (lukekim) [inline rds-aurora-mysql/README.md]: Suffix: Data Connector

File: rds-postgresql/README.md [hunk 10]
```diff
@@ -1,6 +1,8 @@
...
+# AWS RDS for PostgreSQL
```
Comment (lukekim) [inline rds-postgresql/README.md]: Suffix: Data Connector

File: README.md [hunk 11]
```diff
@@ -14,8 +14,9 @@ Designed to take 1-5 mins each to get started quickly!
 
 - [S3 Data Connector](./s3/README.md)
 - [Databricks Delta Lake Data Connector](./databricks/README.md)
-- [Supabase Data Connector](./supabase/README.md)
-- [AWS RDS Data Connector](./rds/README.md)
+- [Supabase Data Connector Data Connector](./supabase/README.md)
+- [AWS RDS PostgreSQL Data Connector](./rds-postgresql/README.md)
+- [AWS RDS Aurora (MySQL Compatible)](./rds-aurora-mysql/README.md)
```
Comment (lukekim) [inline README.md]: Data Connector suffix

**Review by lukekim — DISMISSED** [review]

**Review by digadeesh — DISMISSED** [review]

File: README.md (hunk 11 above)
Comment (ewgenius) [reply README.md]: oh I missed that, thanks

**Review by digadeesh — APPROVED** [review]

**Pull request merged**